programmes/static/css/manifesto.scss

```diff
@@ -6,6 +6,28 @@
   padding: 3rem 0;
 }
 
+.manifesto-menu {
```

Comment:
Font size could be smaller

Reply:
what size would you like, I neet to import a new google font for that ?

Reply:
No new font needed, just font-size: .75rem or something like that